modules/contracts/adapters/cw-staking/src/handlers/execute.rs

```diff
@@ -52,14 +53,15 @@ fn handle_local_request(
 }
 
 /// Handle a request that needs to be executed on a remote chain
+/// TODO, this doesn't work as is. This should be corrected when working with ibc hooks ?
 fn handle_ibc_request(
```

Comment:
How do we see IBC requests being managed by these adapters?

IMO the chain that the user would like to use should be provided by the user (kinda like it is right now). 
Thoughts?